src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -44,35 +44,33 @@ export default function StatusCard({ sightingData }) {
   const photoCount = get(sightingData, ['assets', 'length'], 0);
   const dateCreated = get(sightingData, 'createdHouston');
 
-  const detectionComplete =
-    get(sightingData, 'stage') !== 'detection';
-  const curationComplete =
-    detectionComplete && get(sightingData, 'stage') !== 'curation';
-  const matchingComplete =
-    curationComplete && get(sightingData, 'stage') !== 'un_reviewed';
+  const detectionStartTime = sightingData?.detection_start_time;
+  const curationStartTime = sightingData?.curation_start_time;
+  const matchingStartTime = sightingData?.identification_start_time;
+  const sightingStage = sightingData?.stage;
 
-  const detectionStartTime = get(
-    sightingData,
-    'detection_start_time',
-  );
-  const curationStartTime = get(sightingData, 'curation_start_time');
-  const matchingStartTime = get(
-    sightingData,
-    'identification_start_time',
-  );
+  const detectionComplete = detectionStartTime && curationStartTime;
+  const curationComplete = detectionComplete && matchingStartTime;
+  const matchingComplete =
+    curationComplete && sightingStage !== 'reviewed';
 
   let detectionStage = stages.waiting;
   let curationStage = stages.waiting;
   let matchingStage = stages.waiting;
 
-  if (detectionStartTime) detectionStage = stages.current;
+  if (detectionStartTime) {
+    detectionStage =
+      sightingStage === 'failed' ? stages.failed : stages.current;
```

Comment:
More a cosmetic comment than anything:
sightingStage here seems like it will be easily confused with stages.
Maybe calling it sightingStatus instead would be more informative?
I dunno. Feel free to disregard. I had to read this a few times, though.

Reply:
Also, `sightingStage === 'failed' ? stages.failed : stages.current;` gets repeated several times below. 
I sense an opportunity to DRY.

Reply:
changed the name! not gonna bother with DRY right now since it's just one line repeated a couple times, but if it gets worse i'll address that too